yarn.lock

```diff
@@ -16378,7 +16386,7 @@ ieee754@^1.1.12, ieee754@^1.1.4:
   resolved "https://registry.yarnpkg.com/ieee754/-/ieee754-1.1.13.tgz#ec168558e95aa181fd87d37f55c32bbcb6708b84"
   integrity sha512-4vf7I2LYV/HaWerSo3XmlMkp5eZ83i+/CDluXi/IGTs/O1sejBNhTtnxzmRZfvOUqj7lZjqHkeTvpgSFDlWZTg==
 
-ieee754@^1.1.13:
+ieee754@^1.1.13, ieee754@^1.2.1:
```

Comment:
__nit:__ looks like you can merge this declaration with another `ieee754` declaration above.

Reply:
Ah, haven't noticed enabled auto-merge :cry:

Reply:
Ah good catch. I created a follow up PR: #129159